deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/impl/V1PodHandler.java

```diff
@@ -372,4 +386,190 @@ public boolean cleanUp() {
 	public int getWeight() {
 	    return ResourceKind.POD.getWeight();
 	}
+	
+	/**
+     * It will watch event changes in the pods from cluster based on namespace, appname and service name.
```

Comment:
We need to add more details like fetching pod parent, based on pod parent let updated selector, then the selector is used to watch pods.
While watching pods what all activity are considered also needs to be added